packages/slice-machine/lib/builders/CustomTypeBuilder/TabModal/update.tsx

```diff
@@ -86,77 +44,44 @@ const CreateCustomtypeForm = ({
         }
       }}
       content={{
-        title,
+        title: "Update Tab",
```

Comment:
⚠️ I know it's not yet the new modal but Since the label is Rename should be put Rename Tab, before Edit / Update was for both action, now it's only Renaming.